Modules/Nonunit/Review/include/itkLabelGeometryImageFilter.h

```diff
@@ -465,7 +465,7 @@ class ITK_TEMPLATE_EXPORT LabelGeometryImageFilter:
 
 protected:
   LabelGeometryImageFilter();
-  ~LabelGeometryImageFilter() override{}
+  ~LabelGeometryImageFilter() override= default;
```

Comment:
It would be nice if you could add a space character after `override`. (No showstopper, though.)